coralproject/talk
# [CORL-2409]: label flags in moderate card

## What does this PR do?

This adds a label of "Latest reports" to the flags that have been reported for a comment in its moderate card in moderation queues.

It also migrates over all `regularQueue.spec.tsx` tests to RTL.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Go to a comment with flags in the moderation queue. Click `Details`. See the flags for the comment. See that they are labeled with `Latest reports`.
 
## How do we deploy this PR?
